.github/workflows/publish.yml

```diff
@@ -61,3 +54,11 @@ jobs:
         registry: docker.io/grocy
         username: ${{ secrets.DOCKERHUB_USERNAME }}
         password: ${{ secrets.DOCKERHUB_TOKEN }}
+    # Perform vulnerability scanning
+    ## Note: please move this to *before* the container publish steps after confirming a way to use anchore/scan-action with locally-built OCI images
+    - uses: anchore/scan-action@v3
+      with:
+        image: docker.io/grocy/${{ steps.build-grocy-backend.outputs.image-with-tag }}
+    - uses: anchore/scan-action@v3
+      with:
+        image: docker.io/grocy/${{ steps.build-grocy-frontend.outputs.image-with-tag }}
```

Comment:
Note: could have added `push-to-registry` step IDs, and then used `${{ steps.<id-for-push-to-registry-step>.outputs.registry-path }}` ([doc ref](https://github.com/redhat-actions/push-to-registry/blob/df878026e36864aa6bf9a5b7afe63637cb88ef8c/README.md#action-outputs)) -- but haven't done here, because in theory these two actions should be cut-and-paste relocatable in future if `scan-action` supports local OCI scanning in v3.